src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h

```diff
@@ -39,12 +39,8 @@
 #endif
 
 #ifndef _WIN32
-//#include <boost/algorithm/string.hpp>
-#ifdef __GLIBC__
-#include <xlocale.h>
 #include <sys/time.h>
```

Comment:
This will break Alpine Linux, please keep `#ifdef __GLIBC__`.

Can Alpine job be added to this repo?

Reply:
@kasper3 We have an Alpine job in this repo. I was also wondering why was it not running at first, but then I've realized that this change is in release/1.1 branch and that branch doesn't support Alpine at all. So the change is ok as it is, since this will never be compiled on Alpine.
The first .NET core supporting Alpine is 2.1.

Reply:
Yeah there aren't any plans to add Alpine support to 1.1

Also, @kasper3 why does the Alpine build need xlocale.h? newer glibc versions don't expose it, so you'll run into problems there.

Reply:
@ianhays Alpine is not based on glibc, but rather on MUSL.

Reply:
It's about `sys/time.h` header which is not available in Alpine. I think `sys/*` headers aren't available in Alpine as they are either without `sys` prefix or implied by default like `sys/sysctl` etc.
Check for non-glibc using `#ifdef` is mostly for Alpine and Android (Android's using bionic-libc).

Reply:
In that case, shouldn't we have a glibc check in master?

https://github.com/dotnet/core-setup/blob/master/src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h#L41

Reply:
@ianhays https://github.com/dotnet/core-setup/commit/96d73785528eca8914b40741b44d96243546706e#diff-bbf8763b0760b6908e10eef725465d68 explicitly moved `sys/time.h` out of the glibc guard :confused:

Reply:
@ianhays, my bad, it was only `<sys/sysctl.h>` not present in musl. I should have checked before. Sorry for the noise.
@omajid, this PR is targeting 1.1 branch, which doesn't support Alpine (or any non-glibc Unix platforms)

Reply:
@kasper3 no worries, just wanted to make sure we were doing the right thing in master :)